server/services/caService.js

```diff
@@ -30,9 +31,10 @@ module.exports = function createCaService(roService, lduActiveClient, { continue
           `Found reason for not continuing processing booking: ${bookingId}, error: ${error.code}:${error.message}`
         )
         switch (error.code) {
-          case NO_OFFENDER_NUMBER:
-          case NO_COM_ASSIGNED:
-            return error.code
+          case 'NO_OFFENDER_NUMBER':
+            return { NO_OFFENDER_NUMBER }
+          case 'NO_COM_ASSIGNED':
+            return { NO_COM_ASSIGNED }
```

Comment:
Not sure why this has changed? why return objects?

Reply:
ok

Reply:
No - your approach of returning logical codes rather than content seems better.

Reply:
done